experimental/cmd/mfrc522/main.go

```diff
@@ -0,0 +1,131 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"periph.io/x/periph/conn/spi/spireg"
+	"periph.io/x/periph/experimental/devices/mfrc522"
+	"periph.io/x/periph/experimental/devices/mfrc522/commands"
+	"periph.io/x/periph/host"
+	"strconv"
+	"strings"
+)
+
+func mainImpl() (err error) {
+
+	sector := flag.Int("sector", 1, "Sector to access")
+	block := flag.Int("block", 0, "Block to access")
+
+	rsPin := flag.String("rs-pin", "", "Reset pin")
+	irqPin := flag.String("irq-pin", "", "IRQ pin")
+
+	keyCommand := flag.Bool("wa", false, "Overwrite keys")
+	blockCommand := flag.Bool("wb", false, "Overwrite block by [0-15]")
+
+	spiID := flag.String("spi", "", "SPI device")
+
+	key := flag.String("key", "", "Comma-separated key bytes")
+
+	flag.Parse()
+
+	if *irqPin == "" || *rsPin == "" {
+		return errors.New("please provide -rs-pin and -irq-pin arguments, or -h for help")
+	}
+
+	if _, err = host.Init(); err != nil {
+		return
+	}
+
+	currentAccessKey := mfrc522.DefaultKey
+
+	if *key != "" {
+		keyBytes := strings.SplitN(*key, ",", 6)
+		if len(keyBytes) != 6 {
+			return errors.New("key should consist of 6 decimal numbers")
+		}
+		currentAccessKey = make([]byte, 6)
+
+		for i, v := range keyBytes {
+			intV, err := strconv.Atoi(v)
```

Comment:
You should also make sure this doesn't overflow. The best way is to use https://golang.org/pkg/strconv/#ParseUint and pass 8 as bitSize.